lib/src/model/package_builder.dart

```diff
@@ -23,15 +23,17 @@ import 'package:analyzer/src/generated/source_io.dart';
 import 'package:analyzer/src/source/package_map_resolver.dart';
 import 'package:dartdoc/src/dartdoc_options.dart';
 import 'package:dartdoc/src/logging.dart';
-import 'package:dartdoc/src/model/model.dart';
+import 'package:dartdoc/src/model/model.dart' hide Package;
 import 'package:dartdoc/src/quiver.dart' as quiver;
 import 'package:dartdoc/src/package_config_provider.dart';
 import 'package:dartdoc/src/package_meta.dart'
     show PackageMeta, PackageMetaProvider;
 import 'package:dartdoc/src/render/renderer_factory.dart';
 import 'package:dartdoc/src/special_elements.dart';
 import 'package:meta/meta.dart';
+// TODO(jcollins-g): do not directly import path, use ResourceProvider instead
```

Comment:
Ooh good call. I had an idea in my head to grep for path imports and clean them up.